CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 * **[Fix]** Fix `isExpired` method in `LocalDocument` for incorrect handling of the `TimeToLive.INFINITE` value. 
 * **[Feature]** Add `ReadOptions` parameter to the `list` API.
 * **[Feature]** Serialize `null` document values.
+* **[Fix]** Fix declaring `gson` as a build time dependency instead of runtime.
```

Comment:
Not blocking and not your change. We need to group it together by Fix and Feature. L15.